src/PrestaShopBundle/Controller/Admin/Configure/AdvancedParameters/WebserviceController.php

```diff
@@ -0,0 +1,180 @@
+                $submittedForm =$request->request->get('form');
+                if (array_key_exists('webservice_configuration', $submittedForm)) {
+                    return $this->processFormAction($request);
+                }
+            }
+        }
+
+        $form = is_null($form) ? $this->getFormHandler()->getForm() : $form;
+
+        $presentedGrid = $this->handleListingSearch($request);
+
+        $configurationWarnings = $this->lookForWarnings($request);
+
+        if (false === empty($configurationWarnings)) {
+            foreach ($configurationWarnings as $warningMessage) {
+                $this->addFlash('warning', $warningMessage);
+
+            }
+        }
+
+        $twigValues = [
+            'layoutTitle' => $this->trans('Webservice', 'Admin.Navigation.Menu'),
+            'requireAddonsSearch' => false,
+            'requireBulkActions' => false, // temporary
+            'showContentHeader' => true,
+            'enableSidebar' => true,
+            'help_link' => $this->generateSidebarLink('AdminWebservice'),
+            'requireFilterStatus' => false,
+            'form' => $form->createView(),
+            'grid' => $presentedGrid,
+        ];
+
+        return $this->render('@AdvancedParameters/WebservicePage/webservice.html.twig', $twigValues);
+
+    }
+
+    /**
+     * Handles the webservice configuration form
+     *
+     * @param Request $request
+     *
+     * @return RedirectResponse
+     */
+    private function processFormAction(Request $request)
+    {
+        $this->dispatchHook('actionAdminAdminWebserviceControllerPostProcessBefore', array('controller' => $this));
+
+        $form = $this->getFormHandler()->getForm();
+        $form->handleRequest($request);
+
+        if ($form->isSubmitted()) {
+            $data = $form->getData();
+
+            $saveErrors = $this->getFormHandler()->save($data);
+
+            if (0 === count($saveErrors)) {
+                $this->addFlash('success', $this->trans('Update successful', 'Admin.Notifications.Success'));
+
+                return $this->redirectToRoute('admin_webservice');
+            }
+
+            $this->flashErrors($saveErrors);
+        }
+
+        return $this->redirectToRoute('admin_webservice');
+    }
+
+    /**
+     * @param Request $request
+     *
+     * @return string[]
+     */
+    private function lookForWarnings(Request $request)
+    {
+        /** @var WebserviceCanBeEnabledConfigurationChecker $configurationChecker */
+        $configurationChecker = $this->get('prestashop.core.configuration.webservice_can_be_enabled_configuration_checker');
+
+        $warningMessages = $configurationChecker->analyseConfigurationForIssues($request);
+
+        return $warningMessages;
+    }
+
+    /**
+     * @param Request $request
+     *
+     * @return array
+     */
+    private function handleListingSearch(Request $request)
+    {
+        // temporary search criteria class, to be removed ? see TemporarySearchCriteria
```

Comment:
yes, for information you're stuck waiting for thisto be merged  => https://github.com/PrestaShop/PrestaShop/pull/8984